probe-rs/src/probe/stlink/mod.rs

```diff
@@ -1,3 +1,5 @@
+// TODO remove this when ST-Link cleanup by @tiwalun hits!
+#![allow(dead_code, unused_imports)]
```

Comment:
I'd prefer to not add this, otherwise I will forget to remove it again.

Reply:
Hmm, ok, good for me.